specification/dimensions/pricingunit.md

```diff
@@ -0,0 +1,33 @@
+# Pricing Unit
```

Comment:
There have been conversations about having a semi/normalized pricing unit post-1.0 with a separate block size. If we do that, I'm assuming we'll want to keep PricingUnit and UsageUnit aligned in naming and values. Can we rename this to `PricingMeasurement` to keep the provider-specific value separate?

Reply:
Discussed at the meeting and decided to rename the **PricingUnit** column to **PricingMeasure** (non-normalized).